sdk/eventhubs/tests.yml

```diff
@@ -4,21 +4,22 @@ jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: eventhubs
+      TimeoutInMinutes: 120
       EnvVars:
-        AZURE_EVENTHUBS_CONNECTION_STRING: $(java-event-hubs-test-connection-string)
-        AZURE_EVENTHUBS_FULLY_QUALIFIED_DOMAIN_NAME: $(java-event-hubs-fully-qualified-domain-name)
-        AZURE_EVENTHUBS_EVENT_HUB_NAME: $(java-event-hubs-event-hub-name)
+        AZURE_EVENTHUBS_CONNECTION_STRING: $(AZURE_EVENTHUBS_CONNECTION_STRING)
+        AZURE_EVENTHUBS_FULLY_QUALIFIED_DOMAIN_NAME: $(AZURE_EVENTHUBS_FULLY_QUALIFIED_DOMAIN_NAME)
+        AZURE_EVENTHUBS_EVENT_HUB_NAME: $(AZURE_EVENTHUBS_EVENT_HUB_NAME)
```

Comment:
Using the current version of New-TestResources.ps1 (https://github.com/Azure/azure-sdk-for-java/blob/master/eng/New-TestResources.ps1#L242) the variable mapping is no longer required for output variables. If you plumb all of the secrets through the ARM template you can reduce the `EnvVars` section down to: 

```
EnvVars:
    AZURE_TEST_MODE: RECORD
    AZURE_LOG_LEVEL: 4
```

You'll need to plumb the `AZURE_CLIENT_ID` and friends through the ARM template. 

```
...
"parameters": { 
        "tenantId": {
            "type": "string",
            "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
            "metadata": {
                "description": "The tenant ID to which the application and resources belong."
            }
        },
        "testApplicationId": {
            "type": "string",
            "metadata": {
                "description": "The application client ID used to run tests."
            }
        },
        "testApplicationSecret": {
            "type": "string",
            "metadata": {
                "description": "The application client secret used to run tests."
            }
        }, 
...

"outputs": { 
        "AZURE_TENANT_ID": {
            "type": "string",
            "value": "[parameters('tenantId')]"
        },
        "AZURE_CLIENT_ID": {
            "type": "string",
            "value": "[parameters('testApplicationId')]"
        },
        "AZURE_CLIENT_SECRET": {
            "type": "string",
            "value": "[parameters('testApplicationSecret')]"
        },
...
```

Reply:
Update the `EnvVars` section of the tests.yml file: 

```
EnvVars:
    AZURE_TEST_MODE: RECORD
    AZURE_LOG_LEVEL: 4
```

All of the rest of the mappings are redundant now.